.github/workflows/build-image-dispatch.yaml

```
@@ -0,0 +1,28 @@
+name: Build Nightly Image
+
+on:
+  workflow_dispatch:
+    repo: 
```

actionlint (1.7.10)

[error] expected "inputs" key for "workflow_dispatch" section but got "repo"

(syntax-check)